android/assets/jsons/translations/Turkish.properties

```diff
@@ -2001,317 +2001,317 @@ ConditionalsPlacement = before
-Enables conversion of city production to [civWideStat] = 
+Enables conversion of city production to [civWideStat] = Şehir üretiminin [civWideStat] kaynağına dönüştürülmesini sağlar
 Improves movement speed on roads = Birliklerin yollardaki hareket hızını artırır
 Roads connect tiles across rivers = Yollar bölgeleri ırmakların üstünden bağlar
  # Requires translation!
-[relativeAmount]% maintenance on road & railroads = 
+[relativeAmount]% maintenance on road & railroads = Yol ve demiryolu bakım maliyeti [relativeAmount]%
 No Maintenance costs for improvements in [tileFilter] tiles = [tileFilter] Karolar üzerindeki geliştirmeler Bakım ücreti istemez
  # Requires translation!
-[relativeAmount]% construction time for [improvementFilter] improvements = 
+[relativeAmount]% construction time for [improvementFilter] improvements = [improvementFilter] iyileştirmeler için inşaat süresi [relativeAmount]%
  # Requires translation!
-[relativeAmount]% maintenance cost for buildings [cityFilter] = 
+[relativeAmount]% maintenance cost for buildings [cityFilter] = [cityFilter] binaların bakım maliyeti [relativeAmount]%
  # Requires translation!
-Remove [buildingFilter] [cityFilter] = 
+Remove [buildingFilter] [cityFilter] = [cityFilter] şehrinden [buildingFilter] yapısını kaldır
  # Requires translation!
-Sell [buildingFilter] buildings [cityFilter] = 
+Sell [buildingFilter] buildings [cityFilter] = [cityFilter] şehrinde [buildingFilter] yapısını sat
  # Requires translation!
-[relativeAmount]% Culture cost of natural border growth [cityFilter] = 
+[relativeAmount]% Culture cost of natural border growth [cityFilter] = [cityFilter] doğal sınır büyümesinin ♪Kültür maliyeti [relativeAmount]% azalır
  # Requires translation!
-[relativeAmount]% Gold cost of acquiring tiles [cityFilter] = 
-Each city founded increases culture cost of policies [relativeAmount]% less than normal = Kurulan her şehir sosyal politikaların kültür maliyetini normale kıyasla %[relativeAmount] daha az azaltır
+[relativeAmount]% Gold cost of acquiring tiles [cityFilter] = [cityFilter] karo edinme ¤Altın maaliyeti [relativeAmount]% azalır
+Each city founded increases culture cost of policies [relativeAmount]% less than normal = Kurulan her şehir sosyal politikaların ♪Kültür maliyetini normale kıyasla [relativeAmount]% daha az azaltır
  # Requires translation!
-[relativeAmount]% Culture cost of adopting new Policies = 
+[relativeAmount]% Culture cost of adopting new Policies = Yeni politika benimsemenin ♪Kültür maliyeti [relativeAmount]%
  # Requires translation!
-[stats] for every known Natural Wonder = 
+[stats] for every known Natural Wonder = Bilinen her Doğal Harika için [stats]
  # Requires translation!
-[stats] for discovering a Natural Wonder (bonus enhanced to [stats2] if first to discover it) = 
+[stats] for discovering a Natural Wonder (bonus enhanced to [stats2] if first to discover it) = Doğal Harika keşfi için [stats] ¤Altın (İlk keşfeden içim ise [stats2] ¤Altın)
```

Comment:
"Doğal Harika keşfi için [stats] (İlk keşfeden için ise [stats2])" this is better